packages/react/src/primitives/utils/useStableId.ts

```diff
@@ -12,7 +12,7 @@ export type UseStableId = (id?: string) => string;
 // Note: We `toString()` to prevent bundlers from trying to `import { useId } from 'react';`
 // since it doesn't exist in React 17 and below (prevents https://github.com/aws-amplify/amplify-ui/issues/1154)
 const useReactId: () => string | (() => undefined) =
-  (React as any)['useId'.toString()] || (() => undefined);
+  (React['useId'.toString()] as () => string) || (() => undefined);
```

Comment:
Doesn't TS complain that `useId` isn't available in React, or is the `toString` getting around that?